nicolegooden/indecision-maker
# Form / Question view

#### What's this PR do?  
 
- This PR adds functionality to display questions for each activity for the user to interact with
- This PR adds state management for what activities and answers have been selected
- This PR adds the ability to dynamically move between questions when the 'next' button is pressed 

#### Where should the reviewer start? 

- Start by checking for any inconsistencies in the App and Form components

#### How should this be manually tested?  

- Do the pre-explained setup instructions for seeding the database and installing dependencies
- Launch the local server and interact with the application as a user would
- Check that state is accurately being updated each time a user selects an answer

#### Any background context you want to provide?  

- We went through some hiccups here in regard to state management, found that interacting with objects in state is not as easy as it seems.
- We are now at a point where we have access to all the configuration questions that we want to use to determine a random activity
- Next steps will include cleaning up data, styling, fetching the activity data, and building out functional components for each activity

#### What are the relevant tickets?  
closes #19 
closes #24  
closes #22 
closes  #36
closes #37

#### Screenshots (if appropriate) 
<img width="637" alt="Screen Shot 2020-10-30 at 11 52 51 AM" src="https://i.imgur.com/zwyAASC.gif">


#### Questions:
